src/footer/styles.jsx

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@material-ui/core'
+
+export const useStyles = makeStyles({
+  paragraph: { color: 'white' },
+  footer: { backgroundColor: 'blue', height: '10%' },
+  gridHeight: { height: '100%' },
+  divHeight: { height: '90%' },
```

Comment:
There's quite a bit of styling here and for a rather trivial component (I mean, it is only a footer, right?). This seems to me like you've done a bit of over-engineering in here :))). Let's try to change our components to be simpler ( maybe use some Material UI components instead of all these divs, or use other [css units](https://www.w3schools.com/cssref/css_units.asp) as oposed to percentages ). 

If you hit a brick wall, we'll have a pair programming session about it tomorrow :+1:

Reply:
Done